docs/README.md

```diff
@@ -176,6 +177,73 @@ helm upgrade --install aws-ebs-csi-driver \
     aws-ebs-csi-driver/aws-ebs-csi-driver
 ```
 
+##### Upgrading from version 1.X to 2.X of the helm chart
```

Comment:
could you also copy/paste or link this in the new CHANGELOG.md? https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/CHANGELOG.md

(I will edit our github release workflow to link to the CHANGELOG )

Reply:
Done